backend/protos/mgmt/v1alpha1/connection_data.proto

```diff
@@ -66,6 +66,8 @@ message DatabaseColumn {
   string data_type = 4;
   // The isNullable Flag of the column
   string is_nullable = 5;
+  // Indicates where the column has a default value
+  bool has_default = 6;
```

Comment:
I would suggest just returning the default value itself here instead of a computed. Since this is the database schema data, we might actually want to use that in the future. Then this can just be computed easily on the frontend.

```suggestion
  optional string column_default = 6;
```

Reply:
yeah that can work - I was hesitant at first to do that because our front-end really doesn't touch much "actual" data right now, aka data that would be found in a column but the default value might be okay since I guess it's kinda like metadata? so in the future if we do a hybrid deployment, and we want to isolate any data to the backend/worker, then we would maybe have to reverse this to keep the front-end clean of any data. But maybe it's okay for now.